client/src/auth/authActions.js

```diff
@@ -29,6 +29,8 @@ export function authenticateUser(email, password) {
     dispatch(push('/profile'));
   })
   .catch(() => {
+    console.log('Error in login.');
```

Comment:
Use console.error, I recommend this format: console.error('Error in login:', err)